### DISCUSSION TITLE
Preset for "Forced Overwrite of Sampling Step"

### DISCUSSION THREAD

**Discussion by TheOnlyRealMadman** [lllyasviel/Fooocus#2133] [Q&A] [answered]
Hi,
I built a custom preset which works well but it seems there is no way to change the default setting of "Forced Overwrite of Sampling Step".
Is there a way to do this?
Thank you for your feedback and best regards

**Comment by Irmagon** [comment, marked as answer]
Hi
Try it "default_overwrite_step": 25

**Comment by TheOnlyRealMadman** [comment]
Works fine...thank you.
I tried so many variations and now it turned out the problem was using "25.0" instead of "25"
